lib/File/Find/Rule.pm

```diff
@@ -563,11 +576,15 @@ sub in {
         my $discarded;
         return unless ' . $fragment . ';
         return if $discarded;
-        if ($relative) {
-            push @found, $relpath if $relpath ne "";
+        if ($relative and ($relpath ne "") ) {
+            $normalize
+                ? push(@found, File::Spec->canonpath($relpath))
+                : push(@found, $relpath);
         }
         else {
-            push @found, $path;
+            $normalize
+                ? push(@found, File::Spec->canonpath($path))
+                : push(@found, $path);
```

Comment:
I don't like the use of the ternary operator in void context at all.  Make the code simpler using one of these:

```
my $cleaned_path = $normalize ? File::Spec->canonpath($path) : $path;
push @found, $cleaned_path;
```

Or

```
if ($normalize) {
 push @found, File::Spec->canonpath($path);
} else {
 push @found, $path;
}
```

Or

```
push @found, $normalize ? File::Spec->canonpath($path) : $path;
```

Also don't add the parens to push.  That's not the style the code uses.